react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,41 @@
+label {
+  display: flex;
+  flex-direction: column;
+}
+
+form {
+  display: flex;
+  flex-direction: row;
+}
+
+.input {
+  width: 200px;
+  margin-top: 10px;
+  border: 3px solid black;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  font-family: sans-serif;
+  font-weight: bold;
+}
+
+
+.message {
+  margin-top: 5px;
+  font-family: sans-serif;
+  color: red;
+}
+
+.mark {
+  margin: 30px 0px 0px 5px
+}
+
+.fa-times {
+  color: red;
+}
+
+.fa-check {
+  color: rgb(94, 216, 94);
+}
```

Comment:
You usually don't want to add css rules to existing classes from CSS libraries. This changes the default behavior and can cause issues in other parts of a project. It is better to just create an additional class and add that class to the element. Something like `.text-red`.